build.sbt

```diff
@@ -114,7 +114,9 @@ lazy val `polynote-spark` = project.settings(
       copyRuntimeJar((resourceManaged in Compile).value, "polynote-runtime.jar", (packageBin in (`polynote-runtime`, Compile)).value),
       copyRuntimeJar((resourceManaged in Compile).value, "polynote-spark-runtime.jar", (packageBin in (`polynote-spark-runtime`, Compile)).value)
     )
-  }.taskValue
+  }.taskValue,
+  fork in Test := false,
+  parallelExecution in Test := false
```

Comment:
Did this slow down test execution any?

Reply:
I guess we don't have all that many tests...

Reply:
It actually sped it up in a lot of cases... from infinity to non-infinity 😬 there is some kind of contention when they are run in parallel (at least on travis... can't reproduce it locally)